subsys/net/lib/conn_mgr/events_handler.c

```diff
@@ -39,10 +39,26 @@ static void conn_mgr_iface_events_handler(struct net_mgmt_event_callback *cb,
 	switch (NET_MGMT_GET_COMMAND(mgmt_event)) {
 	case NET_EVENT_IF_CMD_DOWN:
 		iface_states[idx] &= ~NET_STATE_IFACE_UP;
+
+		if (IS_ENABLED(NET_CONNECTION_MANAGER_AUTO_IF_DOWN) &&
```

Comment:
You need to prefix Kconfig entries with `CONFIG_` when used in C, i. e. `CONFIG_NET_CONNECTION_MANAGER_AUTO_IF_DOWN`.

Reply:
🤦My bad -- good catch